api/query/atoms_test.go

```diff
@@ -297,6 +297,34 @@ func TestStructuredQuery_sequential(t *testing.T) {
 			}}}, rq)
 }
 
+func TestStructuredQuery_count(t *testing.T) {
+	var rq RunQuery
+	err := json.Unmarshal([]byte(`{
+		"run_ids": [0, 1, 2],
+		"query": {
```

Comment:
Question outside of this PR - if AbstractExists is the root, should every single query in this test file should start with "exists"?

Reply:
No; Server-side, AbstractExists is only the root when `{exists: ...}` is passed in from the client (which, it always is).